src/main/java/com/paskar/email/application/configuration/SpringConfig.java

```diff
@@ -0,0 +1,32 @@
+package com.paskar.email.application.configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+
+import javax.sql.DataSource;
+
+@Configuration
+public class SpringConfig {
+    @Bean
+    public ObjectMapper mapper() {
+        return new ObjectMapper();
```

Comment:
FYI: org.springframework.http.converter.json.Jackson2ObjectMapperBuilder
I recommend to use this builder.